packages/suite/src/views/wallet/send/components/Outputs/components/Amount/components/Fiat.tsx

```diff
@@ -204,9 +204,9 @@ export const Fiat = ({ output, outputId, labelHoverRight, labelRight }: FiatProp
                 );
 
                 if (updateFiatRatesResult.meta.requestStatus === 'fulfilled') {
-                    const rate = updateFiatRatesResult.payload as number;
+                    const rate = updateFiatRatesResult.payload as fiatRatesResult;
```

Comment:
change name of variable so that it is not used as `rate.rate` or do `const { rate } = updateF...`

Reply:
a1718ae7de5e8b020f8b1c9b763eee8039cc6bd1